builder/scripts.d/20-libiconv.sh

```diff
@@ -5,6 +5,8 @@ SCRIPT_COMMIT="v1.17"
 SCRIPT_TAGFILTER="v?.*"
 
 ffbuild_enabled() {
+    # iconv is macOS built-in
+    [[ $TARGET == mac* ]] && return -1
```

Comment:
I understand what you're doing here, but it's counter-intuitive.
Maybe return 0 directly for `mac*` in `ffbuild_dockerbuild()` and leave `ffbuild_unconfigure()` untouched?